packaging/Containerfile.fedora

```diff
@@ -0,0 +1,19 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+
+COPY rpmbuild/RPMS/x86_64/flightctl-agent-0.0.1-1.el9.x86_64.rpm /tmp/
```

Comment:
Would it make sense to use a COPR repo to host this RPM instead?

Reply:
is the ration work vs benefit worth it? I think having a Makefile target doing the rpm build is enough for now.

Reply:
I think currently a local build would be enough, and helps with easy-quick development cycle.